integration-tests/tests/client/process_blocks.rs

```diff
@@ -1949,8 +1954,23 @@ fn test_block_merkle_proof_with_len(n: NumBlocks) {
             cur_height += 1;
         }
     }
+
     let head = blocks.pop().unwrap();
     let root = head.header().block_merkle_root();
+    // verify that block ordinal to block hash map is correct
+    for h in 0..head.header().height() {
+        if let Ok(block) = env.clients[0].chain.get_block_by_height(h).map(Clone::clone) {
```

Comment:
```suggestion
        if let Ok(block) = env.clients[0].chain.get_block_by_height(h).cloned() {
```

Reply:
This is a `Result`, so `cloned` doesn't work :(